src/current/_data/releases.yml

```diff
@@ -5217,3 +5217,57 @@
     docker_arm_limited_access: false
   source: true
   previous_release: v23.2.0-alpha.7
+
+- release_name: v23.1.13
```

Comment:
This entry is not commented out, so it will show up even though the duplicated one starting in line 5248 will not.